django/common/scripts/cleaning/__init__.py

```diff
@@ -9,6 +9,7 @@
 from .clean_phone import clean_phone
 from .clean_quantity import clean_quantity
 from .clean_time import clean_time
+from .clean_long_text_ORBIS import clean_long_text_ORBIS
```

Comment:
No, I change the function's name in the cleaning script because this cleaning script is for BLOB in Orbis db. We are not sure that this cleaning script will be functional on blob type in others db.

Reply:
Okay great, so you have to remove the other files :)
 that's why the test didn't pass: https://github.com/arkhn/fhir-river/runs/3242780859?check_suite_focus=true#step:6:201

Reply:
(django/common/scripts/cleaning/clean_text_blob.py and the test file)